InstallPackagesArch.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# These are the packages required for install on Arch Linux systems.
+# Taken from the README.
+pacman -Sy gcc qt4 eigen protobuf libdc1394 cmake v4l-utils jsoncpp
```

Comment:
OK, noted. Just as an FYI, this is how it currently stands in the README. I will fix that now.